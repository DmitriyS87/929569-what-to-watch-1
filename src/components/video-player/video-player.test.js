import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from '../video-player/video-payer.jsx';
import VIDEO_PLAYER_MOCK from '../../mocks/test-mocks.js';

const mocks = VIDEO_PLAYER_MOCK.film;

it(`VideoPlayer correctly renders after relunch`, () => {
  const tree = renderer
  .create(<VideoPlayer
    poster={mocks.coverSrc}
    movies={mocks.links}
    title={mocks.title}
  />)
  .toJSON();

  expect(tree).toMatchSnapshot();
});
