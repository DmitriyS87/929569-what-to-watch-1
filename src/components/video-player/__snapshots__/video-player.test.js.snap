// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer correctly renders after relunch 1`] = `
<video
  autoPlay={true}
  controls={true}
  muted={true}
  poster="img/aviator.jpg"
>
  <source
    src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
    type="video/webm"
  />
  <source
    src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv"
    type="video/ogv"
  />
  К сожалению, ваш браузер не поддерживает HTML5 Video.
</video>
`;
